<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowWnd</class>
 <widget class="QMainWindow" name="MainWindowWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doomseeker</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableServers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="menuActionConfigure"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuActionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
